apps/frontend/src/modules/users/users-api.ts

```diff
@@ -20,19 +17,6 @@ class UserApi extends BaseHTTPApi {
 		super({ baseUrl, http, path: APIPath.USERS, storage });
 	}
 
-	public async getAll(): Promise<UserGetAllResponseDto> {
-		const response = await this.load(
-			this.getFullEndpoint(UsersApiPath.ROOT, {}),
```

Comment:
Remove path from UsersApiPath as well

Reply:
If I remove it on the main file, it will affect the mobile side. Should I implement changes on the mobile side as well?